### DISCUSSION TITLE
CellRank v scVelo

### DISCUSSION THREAD

**Discussion by GerardoZA** [theislab/cellrank#963] [Q&A]
Hello, 

I have tried both and seen that both scVelo and CellRank have the same authors. 
Besides the manuscripts, what would say is the major difference between the two? 

Thanks a lot!

**Comment by Marius1311** [comment]
scVelo -> compute RNA velocity & visualize it
CellRank -> use RNA velocity and other biological priors to derive initial & terminal states, fate probabilities and gene expression trends (besides other outputs)
